### PR TITLE
Login Logo was not properly sized

### PR DESCRIPTION
**Pull request by TG908**
Fixed logo view on login screen to look like the one in the CardViewController

**Before**
simulator screen shot - iphone 8 plus - 2018-05-01 at 12 50 04
**After**
simulator screen shot - iphone 8 plus - 2018-05-01 at 12 51 47

### CODE REVIEW

File: TUM Campus App/LoginViewController.swift [hunk 1]
```diff
@@ -91,6 +92,17 @@ class LoginViewController: UIViewController {
...
     }
+    
+    private func setupLogo() {
+        let bundle = Bundle.main
+        let nib = bundle.loadNibNamed("TUMLogoView", owner: nil, options: nil)?.flatMap { $0 as? TUMLogoView }
+        guard let view = nib?.first else { return }
+        logoView = view
+        view.frame = CGRect(x: 0, y: 0, width: 100, height: 40)
+        view.widthAnchor.constraint(equalToConstant: 100).isActive = true
+        view.heightAnchor.constraint(equalToConstant: 40).isActive = true
```
Comment (kordianbruck) [inline TUM Campus App/LoginViewController.swift]: I'm not sure about how this works on iOS, but on Android we have many different resolutions of different phones. Does this automatically scale down&up? From the removed code below, I would guess that would've been the intention. (Wouldn't it be great to have comments to see that the original author thought? :P )

**Comment by thellmund** [comment]
@TG908 I can’t get this to build

**Comment by TG908** [comment]
@thellmund This PR requires #246 to be merged... In #251 these changes are already implemented.

File: TUM Campus App/LoginViewController.swift (hunk 1 above)
Comment (kordianbruck) [reply TUM Campus App/LoginViewController.swift]: @thellmund any comments on this?

Comment (thellmund) [reply TUM Campus App/LoginViewController.swift]: @kordianbruck As far as I know, iOS handles these things like so: The Navigation Bar is _always_ 44 pixels high. On high-res devices, it’s simply drawn at 2x (88 device pixels), and on iPhone X at 3x (132 device pixels). Thus, @TG908’s implementation looks good to me.

**Pull request merged**